### DISCUSSION TITLE
add more info about the forum post on the email notification

### DISCUSSION THREAD

**Discussion by meleu** [RetroAchievements/RAWeb#649] [Ideas]
Once I commented on that "What games you want achievements?" topic I feel like cursed. I'm not interested in following that thread. And, as there's no option to unsubscribe it, I would like to, at least, see the topic's title in the notification message. And then I could delete the message without having to click on the link and being frustrated...

IMO it would save some bandwidth too.

**Comment by Jamiras** [comment]
Would also be nice if the link went straight to the new post instead of having to scroll down through the previous posts that have already been read.

**Comment by Jamiras** [comment]
This concept should also be extended to all notifications: game wall comment, achievement comment, ticket updated, etc.

**Comment by Jamiras** [comment]
#828 added summaries (topic name, achievement title, etc) to the email notifications. It would still be better to include the actual content to avoid having to open the page and scroll to find the changes.

**Comment by Xeinok-Pals** [comment]
Related to discord post: https://discord.com/channels/310192285306454017/1230364875885252608

Totally agree that any post content alert emails should simply contain the content.

**Comment by Jamiras** [comment]
#2725 does this for forum posts and comments.
